test/unit/extract/test_mapping.py

```diff
@@ -182,12 +208,12 @@ def test_build_squash_rules():
     assert actual == ["PATIENTS", [["ADMISSIONS", []]]]
 
 
-def test_find_reference_attributes(patient_pruned):
-    # TODO this test is a bit light
-    fhir_resource = patient_pruned
+# def test_find_reference_attributes(patient_mapping):
```

Comment:
don't you want to add a TODO comment before all the comments related to reference to have a simple way to find what you will have to uncomment later ?